index.html

```diff
@@ -19633,6 +19673,26 @@ <h2 id="MediaStreamTrackAudioSourceNode">
           </li>
         </ul>
       </section>
+      <section>
+        <h2>
+          Unloading a document
+        </h2>
+        <p>
+          When unloading a document with <a>BaseAudioContext</a>s, the
+          following steps MUST be executed:
```

Comment:
This should explicitly say that these are "Unloading document cleanup steps" and link to https://html.spec.whatwg.org/#unloading-document-cleanup-steps